src/bcftools/bcftools_annotate/config.vsh.yaml

```diff
@@ -0,0 +1,251 @@
+      
+      - name: --pair_logic
+        type: string
+        choices: ['snps', 'indels', 'both', 'all', 'some', 'exact']
+        description: | 
+          Controls how to match records from the annotation file to the target VCF. 
+          Effective only when -a is a VCF or BCF file. 
+          The option replaces the former uninuitive --collapse. 
+          See Common Options for more.
+
+      - name: --regions
+        alternatives: --r
+        type: string
+        description: | 
+          Restrict to comma-separated list of regions.
+          See man page for details.
+
+      - name: --regions_file
+        alternatives: --R
+        type: string
+        description: | 
+          Restrict to regions listed in a file.
+          See man page for details.
+
+      - name: --regions_overlap
+        type: string
+        choices: ['pos', 'record', 'variant', '0', '1', '2']
+        description: | 
+          This option controls how overlapping records are determined: 
+          set to pos or 0 if the VCF record has to have POS inside a region (this corresponds to the default behavior of -t/-T); 
+          set to record or 1 if also overlapping records with POS outside a region should be included (this is the default behavior of -r/-R, 
+          and includes indels with POS at the end of a region, which are technically outside the region); 
+          or set to variant or 2 to include only true overlapping variation (compare the full VCF representation "TA>T-" vs the true sequence variation "A>-").
+      
+      - name: --rename_annotations 
+        type: file
+        description: | 
+          Rename annotations: TYPE/old\tnew, where TYPE is one of FILTER,INFO,FORMAT.
+
+      - name: --rename_chromosomes
+        type: file
+        description: | 
+          Rename chromosomes according to the map in file, with "old_name new_name\n" pairs 
+          separated by whitespaces, each on a separate line.
+
+      - name: --samples
+        type: string
+        description: | 
+          Subset of samples to annotate.
+          See also https://samtools.github.io/bcftools/bcftools.html#common_options.
+
+      - name: --samples_file
+        type: file
+        description: | 
+          Subset of samples to annotate in file format.
+          See also https://samtools.github.io/bcftools/bcftools.html#common_options.
+
+      - name: --single_overlaps
+        type: boolean_true
+        description: | 
+          Use this option to keep memory requirements low with very large annotation files. 
+          Note, however, that this comes at a cost, only single overlapping intervals are considered in this mode. 
+          This was the default mode until the commit af6f0c9 (Feb 24 2019).
+
+      - name: --threads
+        type: integer
+        description: | 
+          Number of extra output compression threads.
+          See https://samtools.github.io/bcftools/bcftools.html#common_options.
```

Comment:
can you remove this and pass it from `meta` as `${meta_cpus:+--threads "$meta_cpus"}` instead?
```suggestion
```